J.Coate_H.Nguyen_Case_Study_1.Rmd

```diff
@@ -182,7 +199,12 @@ brew.beer[which.max(brew.beer$IBU),]
 ```{r}
 summary(MedianABV)
 ```
+####
+Huy:
+
+I want to add more to  my last code here. The last code for summary statistics anf distributions on median and per state of ABV and here I want to give a genenral sumary
 
```

Comment:
Why are you giving a general summary when the question only asks for the max IBU? What is the end goal? I think it's really important that we answer the questions we were given then add additional insight. So I think we should definitely leave my code and then add you summary code as a highlight.

Reply:
I agree